src/styles/image.scss

```
@@ -0,0 +1,40 @@
+.image {
+    position: relative;
+    margin: auto;
+    overflow: hidden;
+}
+
+.image-details {
+    position: absolute;
+    bottom: 0%;
+    left: 0%;
+    opacity: 0;
+}
+
+.image:hover .image-details{
+    opacity: 1;
+}
+
+.image-details p {
+    color: #fff;
+    background-color: black;
+    margin-bottom: 0.5em;
+}
+
+.image-icons {
+    position: absolute;
+    top: 0%;
+    left: 0%;
+    z-index: 1;
+    opacity: 0;
+}
+
+.image:hover .image-icons{
+    opacity: 1;
+}
+
+.image-icons i {
+    background-color: #ffffff;
```

The white bleeds out a little from the top and bottom edges of the icons. I'd recommend using a gradient to only cover a part of the icon.
```suggestion
    background-image: radial-gradient(circle, $color-white 60%, transparent 60%);
```